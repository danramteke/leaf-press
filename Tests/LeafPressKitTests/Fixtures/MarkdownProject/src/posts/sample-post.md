published: December 1, 2020
---
#  Markdown Title

Markdown paragraph
