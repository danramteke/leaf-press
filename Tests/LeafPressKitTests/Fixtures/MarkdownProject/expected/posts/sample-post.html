<html>
  <head>
    <link rel="stylesheet" href="/styles.css">
  </head>
  <body>
    <div id="post-content">

<h1>Markdown Title</h1>
<p>Markdown paragraph</p>


    </div>
  </body>
</html>
